Development.md

```diff
@@ -0,0 +1,15 @@
+# Development
+
+## Quick Start
+
+```sh
+git clone https://github.com/ckb-js/kuai.git
+cd kaui
```

Comment:
```suggestion
cd kuai
```

Reply:
The typo `kaui` is not fixed, why is the conversation marked as resolved?

Reply:
I made a mistake and realized that two "kuai" had been typed as "kaui", I applied this suggestion